### PR TITLE
Update "Accuracy" article with ground truth information

### LINKED ISSUE #3430: osu!catch the beat accuracy Formula is wrong
The image which shows the Formula used to calculate the Accuracy in osu!catch the beat is telling some wrong informations. It shows that you need to calculate number of droplets + number of drops + number of fruits divided by every other scorevalues. I implemented this in my code with a recent play request. After that i played a ctb song and compared the results: osu! client said I have 96.04% but with the formula i had 88.99%. I tried editing the Formula by myself and figured out that, if I dont add the missed drops (or countgeki in the api) i reach the 96.04%.

This issure refers to the osu!wiki page [Accuracy](https://osu.ppy.sh/help/wiki/Accuracy/)

### PR DESCRIPTION
**Pull request by bdach**
* Closes #3430 by clarifying what `countGeki` returned from API means for catch.
* Adds a mention about mania using a different accuracy formula with score V2 active.
* Adds info for API users as to where to find 300p and 200 counts for mania in API responses.
* Fixes some tooltip text not matching previous images.
* Replaces all accuracy formula images (as I don't have a way of editing the previous assets).

All information was cross-checked empirically against stable, web API responses, and lazer source.

Previous accuracy formula assets are not removed entirely as they are still used by now-outdated translations.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Gameplay/Accuracy/en.md [hunk 1]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
+- The number of missed droplets is returned as `countKatu`.
+- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
 
 ### ![](/wiki/shared/mode/mania.png) osu!mania
 
-![Accuracy = (50 \* number of 50s + 100 \* number of 100s + 200 \* number of 200s + 300 \* number of 300s + 300 \* number of MAXs) / 300(number of 0s + number of 50s + number of 100s + number of 200s + number of 300s + number of MAXs)](img/accuracy_mania.png "Accuracy formula for osu!mania")
+In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether Score V2 is active.
+
+Without score V2 active, rainbow 300s and gold 300s have an equal weight of 300:
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:48]:
```suggestion
Without ScoreV2 active, rainbow 300s and gold 300s have an equal weight of 300:
```

File: wiki/Gameplay/Accuracy/en.md [hunk 2]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+- The number of caught drops is returned as `count100`.
+- The number of caught droplets is returned as `count50`.
+- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
+- The number of missed droplets is returned as `countKatu`.
+- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
 
 ### ![](/wiki/shared/mode/mania.png) osu!mania
 
-![Accuracy = (50 \* number of 50s + 100 \* number of 100s + 200 \* number of 200s + 300 \* number of 300s + 300 \* number of MAXs) / 300(number of 0s + number of 50s + number of 100s + number of 200s + number of 300s + number of MAXs)](img/accuracy_mania.png "Accuracy formula for osu!mania")
+In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether Score V2 is active.
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:46]:
```suggestion
In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether ScoreV2 is active.
```

File: wiki/Gameplay/Accuracy/en.md [hunk 3]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+
+Without score V2 active, rainbow 300s and gold 300s have an equal weight of 300:
+
+![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania and score V1")
+
+Score V2 increases the weighting of rainbow 300s to 305:
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:52]:
```suggestion
ScoreV2 increases the weighting of rainbow 300s to 305:
```

File: wiki/Gameplay/Accuracy/en.md [hunk 4]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+
+![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and score V2")
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:54]:
```suggestion
![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and ScoreV2")
```

File: wiki/Gameplay/Accuracy/en.md [hunk 5]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
 In osu!catch, accuracy is calculated by taking the total number of non-spinner hit objects collected, divided by the total number of non-spinner objects. All hit objects have the same value; except for bananas, as they are part of the spinner objects.
 
-*Note for API users: To calculate the accuracy in osu!catch, the number of droplets is under `count50` and the number of missed droplets is under `countkatu`.*
+*Notes for API users:*
+
+- The number of caught drops is returned as `count100`.
+- The number of caught droplets is returned as `count50`.
+- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
+- The number of missed droplets is returned as `countKatu`.
+- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
```
Comment (ZeroPyrozen) [inline wiki/Gameplay/Accuracy/en.md:42]: Which API? osu! API v1 or v2 or both?

Comment (bdach) [reply wiki/Gameplay/Accuracy/en.md:42]: Doesn't matter in this context as things stand. `countGeki` and `countKatu` are present in both versions. However I'm pretty sure api v2 will be eventually migrating to something saner at some point.

Comment (ZeroPyrozen) [reply wiki/Gameplay/Accuracy/en.md:42]: Alright add api as hyperlink to this article https://osu.ppy.sh/wiki/en/osu!api

Comment (Walavouchey) [reply wiki/Gameplay/Accuracy/en.md:42]:
```suggestion
*Notes for [API](/wiki/osu!api) users:*

- The number of caught drops is returned as `count100`.
- The number of caught droplets is returned as `count50`.
- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
- The number of missed droplets is returned as `countKatu`.
- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
```

File: wiki/Gameplay/Accuracy/en.md [hunk 6]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether ScoreV2 is active.
+
+Without ScoreV2 active, rainbow 300s and gold 300s have an equal weight of 300:
+
+![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania and score V1")
+
+ScoreV2 increases the weighting of rainbow 300s to 305:
+
+![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania and ScoreV2")
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:54]: programmer moment

```suggestion
![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania with ScoreV2")
```

File: wiki/Gameplay/Accuracy/en.md [hunk 7]
```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
...
+- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
 
 ### ![](/wiki/shared/mode/mania.png) osu!mania
 
-![Accuracy = (50 \* number of 50s + 100 \* number of 100s + 200 \* number of 200s + 300 \* number of 300s + 300 \* number of MAXs) / 300(number of 0s + number of 50s + number of 100s + number of 200s + number of 300s + number of MAXs)](img/accuracy_mania.png "Accuracy formula for osu!mania")
+In osu!mania, accuracy is calculated similarly to [osu!](#osu!). However, the weighting of rainbow 300s (also referred to as MAX results) depends on whether ScoreV2 is active.
+
+Without ScoreV2 active, rainbow 300s and gold 300s have an equal weight of 300:
+
+![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania and score V1")
```
Comment (Walavouchey) [inline wiki/Gameplay/Accuracy/en.md:50]:
```suggestion
![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania with ScoreV1")
```

**Review by Walavouchey — APPROVED** [review]
image naming seems fine for now

**Pull request merged**